CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Head
+
+- Added: `selector-attribute-square-brackets-space-inside` rule.
```

Comment:
As per https://github.com/stylelint/stylelint/issues/1086#issuecomment-212423114, can we rename this `selector-attribute-brackets-space-inside` please?